contrib/babelfishpg_common/src/varchar.c

```diff
@@ -48,9 +48,155 @@ void		TsqlCheckUTF16Length_bpchar(const char *s, int32 len, int32 maxlen, int ch
 void		TsqlCheckUTF16Length_bpchar_input(const char *s, int32 len, int32 maxlen, int charlen);
 void		TsqlCheckUTF16Length_varchar_input(const char *s, int32 len, int32 maxlen);
 static inline int varcharTruelen(VarChar *arg);
+static char *pg_int64tostr(char *str, int64 value);
+static char *pg_int64tostr_zeropad(char *str, int64 value, int64 padding);
 
 #define DEFAULT_LCID 1033
 
+/*
+  * pg_int64tostr Converts 'value' into a decimal string representation of the
+  * number.
+  *
+  * Caller must ensure that 'str' points to enough memory to hold the result
+  * (at least 21 bytes, counting a leading sign and trailing NUL). Return
+  * value is a pointer to the new NUL terminated end of string.
+  */
+static char *
+pg_int64tostr(char *str, int64 value)
```

Comment:
Copied? Have we considered other options like rendezvous variables or moving the code appropriately?
One option could be to call fixeddecimal2str_helper(the OG one from fixeddecimal.c) inside fixeddecimal2varchar either through rendezvous or SQL API

Reply:
Yeah I initially thought that copying these helper functions would be simpler compared to rendezvous or unnecessary SQL function but looks like we can use function pointers for cross module calls. Removed the copied code in the newer revision.